dashboard/src/components/DeploymentForm/DeploymentFormBody/BasicDeploymentForm/TabularSchemaEditorTable/TabularSchemaEditorTableRenderer.tsx

```diff
@@ -0,0 +1,222 @@
+              <></>
+            )}
+          </CdsButton>
+          {renderCellWithTooltip(value, "key", "breakable self-center", true, MAX_LENGTH / 1.5)}
+        </div>
+      </>
+    </div>
+  );
+}
+
+export function renderConfigType(value: IBasicFormParam) {
+  return renderCellWithTooltip(value, "type", "italics");
+}
+
+export function renderConfigDescription(value: IBasicFormParam) {
+  return renderCellWithTooltip(value, "title", "breakable");
+}
+
+export function renderConfigDefaultValue(value: IBasicFormParam) {
+  return renderCellWithTooltip(value, "defaultValue", "breakable");
+}
+
+export function renderConfigDeployedValue(value: IBasicFormParam) {
+  return renderCellWithTooltip(value, "deployedValue");
+}
+
+export function renderConfigCurrentValuePro(
+  param: IBasicFormParam,
+  handleBasicFormParamChange: (
+    p: IBasicFormParam,
+  ) => (e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement | HTMLSelectElement>) => void,
+) {
+  // early return if the value is marked as a custom form component
+  if (param.isCustomComponent) {
+    // TODO(agamez): consider using a modal window to display the full value
+    return (
+      <div id={param.key}>
+        <CustomFormComponentLoader
+          param={param}
+          handleBasicFormParamChange={handleBasicFormParamChange}
+        />
+      </div>
+    );
+  }
+  // if the param has properties, each of them will be rendered as a row
+  if (param.hasProperties) {
+    return <></>;
+  }
+
+  // if it isn't a custom component or an with more properties, render an input
+  switch (param.type) {
+    case "string":
+      return (
+        <TextParam
+          id={param.key}
+          label={param.title || param.path}
+          param={param}
+          inputType={
+            [param.title, param.key].some(s => s.match(/password/i)) ? "password" : "string"
+          }
+          handleBasicFormParamChange={handleBasicFormParamChange}
+        />
+      );
+
+    case "boolean":
+      return (
+        <BooleanParam
+          id={param.key}
+          label={param.title || param.path}
+          param={param}
+          handleBasicFormParamChange={handleBasicFormParamChange}
+        />
+      );
+
+    case "integer":
+    case "number":
+      return (
+        <SliderParam
+          id={param.key}
+          label={param.title || param.path}
+          param={param}
+          handleBasicFormParamChange={handleBasicFormParamChange}
+          step={param.type === "integer" ? 1 : 0.1}
+          unit={""}
+        />
+      );
+    case "array":
+      if (param?.schema?.items?.type !== "object") {
+        return (
+          <ArrayParam
+            id={param.key}
+            label={param.title || param.path}
+            param={param}
+            handleBasicFormParamChange={handleBasicFormParamChange}
+            type={param?.schema?.items?.type ?? "string"}
+          />
+        );
+      } else {
+        // TODO(agamez): render the object properties
```

Comment:
Leaving this case explicit as we might want to render a more user-friendly editor for objects (like "+ Add property" button or sth like this).